editor/project_settings_editor.cpp

```diff
@@ -178,15 +178,15 @@ void ProjectSettingsEditor::_action_edited() {
 	}
 
 	int order = ProjectSettings::get_singleton()->get_order(add_at);
-	Array va = ProjectSettings::get_singleton()->get(add_at);
+	Dictionary action = ProjectSettings::get_singleton()->get(add_at);
```

Comment:
Why dont you just get a variant and check the type of variant? you can abstract this probably in a single function that returns a Dictionary